apiary.apib

```diff
@@ -997,6 +997,143 @@ Outputs the list of peers that were removed.
 
 # Group ls
 
+List links from an object.
```

Comment:
This is kind of an alias to `ipfs object links` (which I believe historically came after), it is one of those that live in the limbo now that we have a `files` command for anything that is files/unixFS (ipfs object links would print the same links because the links in a UnixFS proto are the ones that are stored in the MerkleDAG proto)